src/test/regress/expected/prepared_statements_4.out

```diff
@@ -0,0 +1,52 @@
+SET search_path TO "prepared statements";
+PREPARE prepared_relabel_insert(varchar) AS
+	INSERT INTO text_partition_column_table VALUES ($1, 1);
+EXECUTE prepared_relabel_insert('test');
+EXECUTE prepared_relabel_insert('test');
+EXECUTE prepared_relabel_insert('test');
+EXECUTE prepared_relabel_insert('test');
+EXECUTE prepared_relabel_insert('test');
+EXECUTE prepared_relabel_insert('test');
+EXECUTE prepared_relabel_insert('test');
+SELECT key, value FROM text_partition_column_table ORDER BY key;
+ key  | value 
+------+-------
+ test |     1
+ test |     1
+ test |     1
+ test |     1
+ test |     1
+ test |     1
+ test |     1
+(7 rows)
+
+PREPARE prepared_coercion_to_domain_insert(text) AS
+	INSERT INTO domain_partition_column_table VALUES ($1, 1);
+EXECUTE prepared_coercion_to_domain_insert('test-1');
```

Comment:
How easy would it be to add a utility function that calls the given method 7 times to use the prepared statement cache instead of always writing the same command 7 times?

Reply:
Well, these 7 repetition is simple to follow, right? I think we can add, but feels over engineered for such a thing?

Reply:
I thought that it could be useful in general in the test suite because of 2 main reasons:
- When someone sees 7 repetitions, it might not be clear why we have this, we sometimes add a comment to explain this but sometimes not. If we had a utility method for this, we could write a longer comment and then everyone can read the longer comment to better understand this.
- This might reduce the #lines in the test so the test file would look nicer.

I am not saying we should add right now, but let me know what you think about the above points